notebook-src/02-book-balancing-intro.rst

```diff
@@ -0,0 +1,170 @@
+======================================================
+2. Introduction to vibrations: Book Balancing on a Cup
+======================================================
+
+This notebook introduces a single degree of freedom vibratory system in which a
+textbook balances on a cylindrical cup. The system is implemented as a model
+that students can interact with in order to visualize its free response and
+compare to the demonstration in the classroom.
+
+After the completion of this assignment students will be able to:
+
+- visualize a system's free response
+- estimate the period of a sinusoidal vibration from a time series
+- compare a computer simulation result to experimental result
+- interactively adjust the book inertia to see the affect on system response
+- understand the concept of natural frequency nd its relationship to
+  mass/inertia
+
+What are vibrations?
+
+Here we will study a simple vibratory system. If you set a cylindrical cup on
+its side and place a book on top of it the book will oscillate if initialized
+at a small angle non-horizontal angle or it is perturbed slightly. Note that it
+oscillates about an equilibrium point, the rest angle defines this equilibrium
+point. Vibration is defined as an oscialation about an equilibrium.
+
+During this class we will examine and explore many different vibrating systems,
+such as this simple book and cup system. We will have some live demos as we are
+showing now but in general we will work with computational systems to
+experiment and learn about a system. For example, here is a system that
+represents the book on a cup::
+
+   >>> from resonance.linearsystems import BookOnCupSystem
+
+The line above loads the command that can create systems that behave like the
+live demo at teh fron of the room. To create a system::
+
+   >>> sys = BookOnCupSystem()
```

Comment:
We should have a way that the students don't have to subclass system, but just add a the eoms. We don't want them to have to learn about classes.